include/Sound.h

```diff
@@ -1,35 +1,28 @@
 #ifndef SOUND_H
 #define SOUND_H
 
-#include <string>
-#include <iostream>
-
 #include <fmod.hpp>
-#include "fmod_errors.h"
 
 class Sound{
 
-public:
-
-  bool playing;
-
-  FMOD_RESULT result;
-
-  FMOD::Channel *channel = NULL;
-
-  FMOD::Sound *audio;
-
-  FMOD::System *m_pSystem;
-
-  Sound();
-
-  void createSound( const char*);
```

Comment:
@ma302jh Although you can declare a function's parameters like this, it helps to name your parameters. For example I know that the function createSound takes in a const char* but what is the const char* supposed to be? A song file, raw data of the song file, the song name? See below where I named the parameter to track_dir.